apps/nextjs/src/components/ContextProviders/ChatProvider.tsx

```diff
@@ -245,6 +249,50 @@ export function ChatProvider({ id, children }: Readonly<ChatProviderProps>) {
       messageHashes,
     });
 
+  // Handle queued user actions and messages
```

Comment:
Ideally, rather than adding to this function this logic could be in its own hook called by the provider. I know that there's time pressure for this release though